### PR TITLE
New documentation section for certificates information indicators

### PR DESCRIPTION
**Pull request by leandroberetta**
This PR adds a new section in the features page about the feature certificates information indicators.

### CODE REVIEW

File: content/documentation/v1.39/features/index.adoc [hunk 1]
```diff
@@ -862,6 +862,30 @@ Regarding the *severity* of each component, there are only to options: `core` or
 
 By default, Kiali checks the following components installed in the control plane namespace: istiod, ingress, egress; and prometheus, grafana and jaeger accessible thought their services.
 
+== Certificates Information Indicators
+
+Kiali v1.40 introduces a new section in the Help menu for viewing certificates used by internal mTLS.
```
[the remark this answers is not in the document]
Comment (leandroberetta) [reply content/documentation/v1.39/features/index.adoc]: Thanks for your comments!. Version removed, moved to staging.

**Comment by jmazzitelli** [comment]
To make kiali.io PRs easier to review, you should post the netlify URLs when they are ready so people can click on the links and go directly to the doc pages.

**Review by xeviknal — APPROVED** [review]
LGTM!

**Comment by jmazzitelli** [comment]
I believe the two netlify links for this PR are:

* https://deploy-preview-407--kiali.netlify.app/documentation/staging/installation-guide/#_updating_permissions
* https://deploy-preview-407--kiali.netlify.app/documentation/staging/features/#_certificates_information_indicators

File: content/documentation/staging/features/index.adoc [hunk 2]
```diff
@@ -862,6 +862,35 @@ Regarding the *severity* of each component, there are only to options: `core` or
 
 By default, Kiali checks the following components installed in the control plane namespace: istiod, ingress, egress; and prometheus, grafana and jaeger accessible thought their services.
 
+== Certificates Information Indicators
+
+In some situations, it would be useful to get information about the certificates used by internal mTLS, for example:
+
+* Know whether the default CA is used or if there is other CA configured
```
Comment (jmazzitelli) [inline content/documentation/staging/features/index.adoc:869]:
```suggestion
* Know whether the default CA is used or if there is another CA configured
```

File: content/documentation/staging/features/index.adoc [hunk 3]
```diff
@@ -862,6 +862,35 @@ Regarding the *severity* of each component, there are only to options: `core` or
...
+* Check the certificates issuer and their validity timestamps to troubleshoot any issue with certificates
+
+The certificates shown depends on how Istio is configured, the following cases are possible:
```
Comment (jmazzitelli) [inline content/documentation/staging/features/index.adoc:872]:
```suggestion
The certificates shown depends on how Istio is configured. The following cases are possible:
```

File: content/documentation/staging/features/index.adoc [hunk 4]
```diff
@@ -862,6 +862,35 @@ Regarding the *severity* of each component, there are only to options: `core` or
...
+
+* Using Istio CA certificates (default), the information shown is got from a secret named *istio-ca-secret*
+* Using https://istio.io/latest/docs/tasks/security/cert-management/plugin-ca-cert/[Plug in CA certificates, window=_blank], the information shown is got from a secret named *cacerts*
+* Using https://istio.io/latest/docs/tasks/security/cert-management/dns-cert/[DNS certificates, window=_blank], the information shown is got by reading many secrets found in Istio configuration
```
Comment (jmazzitelli) [inline content/documentation/staging/features/index.adoc:876]:
```suggestion
* Using Istio CA certificates (default), the information shown is from a secret named *istio-ca-secret*
* Using https://istio.io/latest/docs/tasks/security/cert-management/plugin-ca-cert/[Plug in CA certificates, window=_blank], the information shown is from a secret named *cacerts*
* Using https://istio.io/latest/docs/tasks/security/cert-management/dns-cert/[DNS certificates, window=_blank], the information shown is from reading many secrets found in Istio configuration
```

File: content/documentation/staging/features/index.adoc [hunk 5]
```diff
@@ -862,6 +862,35 @@ Regarding the *severity* of each component, there are only to options: `core` or
...
+<div style="display: flex;">
+ <span style="margin: 0 auto;">
+  <a class="image-popup-fit-height" href="/images/documentation/features/certificates-information-indicators.png" title="Certificates information">
+   <img src="/images/documentation/features/certificates-information-indicators.png" style="width: 1333px;display:inline;margin: 0 0 15px 0;" />
+  </a>
+ </span>
+</div>
+++++
+
+Note that displaying this configuration requires permissions to read secrets (istio-ca-secret by default, possibly cacerts or any secret configured when using DNS certificates).
```
Comment (jmazzitelli) [inline content/documentation/staging/features/index.adoc:890]:
```suggestion
Note that displaying this configuration requires permissions to read secrets (*istio-ca-secret* by default, possibly *cacerts* or any secret configured when using DNS certificates). 
```

File: content/documentation/staging/installation-guide/index.adoc [hunk 6]
```diff
@@ -509,6 +515,44 @@ deployment:
...
+When installing the operator using the Helm chart, set the property `secretReader`. Secrets listed in this property will be allowed to be read by the operator and subsequently by Kiali.
+
+By default the secrets listed are:
+
+[source,yaml]
+----
+secretReader: ['cacerts', 'istio-ca-secret']
+----
+
+When installing the operator with OLM there is no way to update the operator permissions after installing it, the solution is to manually update the cluster role used by the operator as follows:
```
Comment (jmazzitelli) [inline content/documentation/staging/installation-guide/index.adoc:548]: This is contradictory - first it says "there is no way to update the operator permissions after installing it" and then the following instructions tell you how to update the operator permissions after you installed it. :-)

I think we want this to say:

```suggestion
When installing the operator with OLM there is no way to pre-configure the operator permissions prior to installing the operator. The solution is to manually update the cluster role used by the operator as follows:
```

**Review by jmazzitelli — CHANGES_REQUESTED** [review]
Just some minor tweeks

**Comment by leandroberetta** [comment]
@xeviknal @jmazzitelli thanks for the review. All suggestions committed.

**Review by jmazzitelli — APPROVED** [review]

**Comment by jmazzitelli** [comment]
Ignore the verification error - looks like redhat.com took too long when netlify was trying to verify a link:
```
  *  External link https://www.redhat.com/en/visualize_and_secure_your_service_mess_with_rh_os_service_mesh failed: got a time out (response code 0)
```

**Comment by abonas** [comment]
> To make kiali.io PRs easier to review, you should post the netlify URLs when they are ready so people can click on the links and go directly to the doc pages.

aren't those posted as part of the checks when CI is built?
this part:
netlify/kiali/deploy-preview — Deploy Preview ready!

**Comment by jmazzitelli** [comment]
> aren't those posted as part of the checks when CI is built?
> this part:
> netlify/kiali/deploy-preview — Deploy Preview ready!

Click that link - then try to find the docs that this PR changed :) You will see what I mean.

It's much easier just to post a link directly to the staging page/section where the changes are.

**Comment by leandroberetta** [comment]
@abonas @jmazzitelli thanks for the advice, I will include the links next time!.

**Pull request merged**